astropy/table/table_helpers.py

```diff
@@ -105,7 +105,7 @@ def simple_table(size=3, cols=None, kinds="ifS", masked=False):
         elif kind == "O":
             indices = (np.arange(size) + jj) % len(letters)
             vals = letters[indices]
-            data = [{val: index} for val, index in zip(vals, indices)]
+            data = [{val.item(): index.item()} for val, index in zip(vals, indices)]
```

Comment:
Can you confirm this function looks is *only* meant for testing and simple doc examples ?

Reply:
Yes, confirmed!